### PR TITLE
Update docs on NPROC

### PR DESCRIPTION
**Pull request by sandstrom**
This would have saved me 5 min. Thought I'd help people coming after me.

Haven't checked if RPM packages does this too, but if so we can add it.

This change is trivial, so no CLA signature is required.

### CODE REVIEW

File: docs/reference/setup/sysconfig/threads.asciidoc [hunk 1]
```diff
@@ -10,3 +10,4 @@ This can be done by setting <<ulimit,`ulimit -u 4096`>> as root before
 starting Elasticsearch, or by setting `nproc` to `4096` in
 <<limits.conf,`/etc/security/limits.conf`>>.
 
+The Debian packages will configure this setting automatically. No further configuration is required.
```
Comment (jasontedor) [inline docs/reference/setup/sysconfig/threads.asciidoc]: This is not quite right, the package distributions only set this automatically against `systemd`.

**Review by jasontedor — CHANGES_REQUESTED** [review]
I left a comment.

File: docs/reference/setup/sysconfig/threads.asciidoc (hunk 1 above)
Comment (sandstrom) [reply docs/reference/setup/sysconfig/threads.asciidoc]: Alright, I've updated the PR

**Review by jasontedor — APPROVED** [review]
Thanks for making this change. I'll push a minor commit to clarify this applies to both package distributions and merge.

**Comment by jasontedor** [comment]
Thanks @sandstrom!

**Pull request merged**